Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN	mkdir -p /usr/src/minetest/cmakebuild && cd /usr/src/minetest/cmakebuild &&
 FROM debian:stretch
 
 USER root
-RUN groupadd minetest && useradd -m -g minetest -d /var/lib/minetest minetest && \
+RUN groupadd -g 30000 minetest && useradd -u 30000 -m -g minetest -d /var/lib/minetest minetest && \
```

Comment:
is there a problem with letting the OS choose these?

Reply:
You don't want the number to match a group on the host

Reply:
the only os who will choose this is the randomness of the image running on gitlab-ci at a moment. We ensure we will always have the same uid/gid across builds, this prevent people who create sqlite3 volumes on /var/lib/minetest mounted on their host to become crazy if the UID changes (and yes this one will change the previous one, which was not stable).
Anyway appart keeping a good default one in the image which should not collide with OS one (i used our default port), it's overwriteable at container startup in any container engine (docker, containerd...)